src/tests/unit/tests/common/components/cards/card-kebab-menu-button.test.tsx

```diff
@@ -12,34 +18,95 @@ describe('CardKebabMenuButtonTest', () => {
 
     const event = {
         currentTarget: 'Card View',
     } as React.MouseEvent<any>;
 
-    const data = 'The quick brown fox jumps over the lazy dog';
+    const issueDetailsText = 'The quick brown fox jumps over the lazy dog';
 
     beforeEach(() => {
+        testIssueFilingServiceStub = {
+            key: testKey,
+            displayName: 'TEST',
+            settingsForm: NamedFC('testForm', () => <>Hello World</>),
+            isSettingsValid: () => true,
+            buildStoreData: testField => {
+                return { testField };
+            },
+            getSettingsFromStoreData: data => data[testKey],
+            fileIssue: () => {},
+        };
         actionCreatorMock = Mock.ofType<DetailsViewActionMessageCreator>();
         navigatorUtilsMock = Mock.ofType<NavigatorUtils>();
+        issueFilingServiceProviderMock = Mock.ofType(IssueFilingServiceProvider);
+        issueFilingActionMessageCreatorMock = Mock.ofType(IssueFilingActionMessageCreator);
+
+        userConfigurationStoreData = {
+            bugService: testKey,
+            bugServicePropertiesMap: {
+                [testKey]: {},
+            },
+            enableHighContrast: false,
+            enableTelemetry: true,
+            isFirstTime: true,
+        };
+
+        issueFilingServiceProviderMock
+            .setup(bp => bp.forKey('test'))
+            .returns(() => testIssueFilingServiceStub)
+            .verifiable();
+
         defaultProps = {
             deps: {
                 detailsViewActionMessageCreator: actionCreatorMock.object,
                 navigatorUtils: navigatorUtilsMock.object,
+                issueFilingServiceProvider: issueFilingServiceProviderMock.object,
+                issueFilingActionMessageCreator: issueFilingActionMessageCreatorMock.object,
+            },
+            userConfigurationStoreData,
+            issueDetailsData: {
+                rule: {
+                    id: 'id',
+                    description: 'description',
+                    url: 'url',
+                    guidance: [
+                        {
+                            href: 'www.test.com',
+                            text: 'text',
+                            tags: [guidanceTags.WCAG_2_1],
+                        },
+                    ],
+                },
+                targetApp: {
+                    name: 'name',
+                    url: 'url',
+                },
+                element: {
+                    identifier: 'identifier',
+                    conciseName: 'conciseName',
+                },
+                howToFixSummary: 'howToFixSummary',
+                snippet: 'snippet',
             },
         } as CardKebabMenuButtonProps;
     });
 
     it('render', () => {
         const rendered = shallow(<CardKebabMenuButton {...defaultProps} />);
 
-        expect(rendered.getElement()).toMatchSnapshot();
+        expect(rendered.debug()).toMatchSnapshot();
```

Comment:
why change to debug()?

Reply:
I used .debug() instead of getElement() because it doesn't work, and that's why I didn't add the expect in the begeining, for more info look here: https://airbnb.io/enzyme/docs/api/ShallowWrapper/debug.html

Reply:
I'm not quite following. Why did it work before but not it's not working now?

Reply:
I don't why it didn't work now, but I know it's expected to work as mentioned in the Debug function definition, if the issue with the function, the debug also should not work, we use the debug in some test cases like start over drop down